src/Microsoft.DotNet.Interactive.DuckDB/DuckDBKernel.cs

```diff
@@ -0,0 +1,193 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+using DuckDB.NET.Data;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.Directives;
+using Microsoft.DotNet.Interactive.Events;
+using Microsoft.DotNet.Interactive.Formatting.TabularData;
+using Microsoft.DotNet.Interactive.ValueSharing;
+
+namespace Microsoft.DotNet.Interactive.DuckDB;
+
+public class DuckDBKernel : Kernel, 
+    IKernelCommandHandler<SubmitCode>,
+    IKernelCommandHandler<RequestValueInfos>,
+    IKernelCommandHandler<RequestValue>
```

Comment:
Thoughts on implementing `SendValue`? 

`#!set` will only work in one direction without it I think.

Reply:
not yes as the declaration of values is not implemented yet. #!set in this kernel has no functionality at the moment